### ISSUE TITLE
lua-resty-coraza production ready?

### ISSUE THREAD

**Issue by victoredvardsson** [potats0/lua-resty-coraza#6]
Hi, im looking into possibly replace lua-resty-lua with lua-resty-coraza.

What is the current plan for this project? And is it ready for production or still in some experimental state?

**Comment by potats0** [comment]
actually, it's ready for production.lua-resty-coraza is a wrapper of the coraza

**Comment by victoredvardsson** [comment]
@potats0 

Yes, it looks good so far. We will probably bring it in to our production env soon. Do you plan to include some more documentation anytime soon?

Like what you can control with the actual lua in nginx.conf.

As an example we have a use case we're we want to keep Coraza in DetectionOnly based on a if statement.

Could probably set different Coraza.conf files based on that but maybe there is some better way?

**Comment by potats0** [comment]
OK, i will write documents ,like how to set DetectionOnly mode in coraza.
Or if you have some demand, you can talk to me. I will improve the lua-resty-coraza.

**Comment by potats0** [comment]
recently, i've developing the waf which is base on openresty with lua, like IP blocker with 100w+ ip/CIDR, url whitelist. Maybe i can help you

**Comment by victoredvardsson** [comment]
Sounds promising, to give you some context. We are currently using this one https://github.com/p0pr0ck5/lua-resty-waf but need to move from it since it's not being developed anymore.

It has some handy features that we use frequently like.

waf:set_option("ignore_rule", 40294)

waf:set_option("mode", inactive)

That's really what I'm looking for at this point.

Thanks for your effort in this!!

**Comment by victoredvardsson** [comment]
@potats0 

Just want to check in and ask about the progress.

Do you have an ETA when documentation/possible new features can be expected?

Thanks!
